### PR TITLE
Do not allow compose command with no features specified.

### PR DESCRIPTION
**Pull request by olafmaurer**
Fixes issue #123.

### CODE REVIEW

File: cli/pazuzu/compose.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ var composeAction = func(c *cli.Context) error {
 		baseImage          string
 	)
 
+    if ((c.String("add") == "")  && (c.String("init") == "")) {
+        return errors.New("Error: No feature specified. Please use at least one of -a or -i for the compose command.")
+    }
+
 	pazuzufilePath, dockerfilePath, err := getAbsoluteFilePaths(c.String("destination"))
```
Comment (rdllopes) [inline cli/pazuzu/compose.go]: why are you not using 'usage' flag on cli  (commands.go)?

Comment (netme) [inline cli/pazuzu/compose.go]: @olafmaurer sidenote: please run `go fmt`, there are some violations :-)

Comment (netme) [inline cli/pazuzu/compose.go]: @olafmaurer please check https://github.com/urfave/cli/blob/master/help.go#L165

**Comment by olafmaurer** [comment]
@rdllopes please review again, I improved it

**Comment by netme** [comment]
👍

**Pull request merged**